swagger.json

```diff
@@ -296,7 +296,33 @@
                     }
                 }
             }
-        }
+        },
+        "/bq/channels": {
```

Comment:
@PedroLabrador this should be just /channels?

Reply:
yes